chainio/clients/avsregistry/reader.go

```diff
@@ -779,6 +783,94 @@ func (r *ChainReader) IsOperatorSetQuorum(
 	return isOperatorSet, nil
 }
 
+func (r *ChainReader) GetPubkeyHashFromOperatorAddress(
```

Comment:
Let's call it operator ID instead of pubkey hash.
```suggestion
func (r *ChainReader) GetOperatorIdFromOperatorAddress(
```

Reply:
fixed